04_Fix-Path.md

```diff
@@ -203,48 +150,49 @@ characters:
     role: Research & Archive
 ```
 
-In the colour example above you see a field `my` which contains a deeper field `colors` which has 3 values. To point to one of the colors you need to use an index but also genuin arrays have a marker in Metafacture: `[]`. Also here the first index in a array has value 1, the second the value 2, the third the value 3. The array markers are generated by the [JSON-Decoder](https://github.com/metafacture/metafacture-documentation/blob/master/flux-commands.md#decode-json) and the [YAML-Decoder](https://github.com/metafacture/metafacture-documentation/blob/master/flux-commands.md#decode-yaml). Also if you want to generate an array in the target format, then you need to add `[]` at the end of an list-element like `newArray[]`. (While sofare the path handling of Catmandu and Metafacture are similar, they differ at this point.)
+Here is another example:
 
-So, the path of the `red` would be: `my.color[].2`
-And the path for `Peter` would be `characters[].2.name`
+```YAML
+my:
+  colors:
+    - black
+    - red
+    - yellow
+```
 
-There is one array type in our JSON report from our example at the beginning above and that is the `weather` field. To point to the description of the weather you need the path `weather[].1.description`.
+To point to one of the colors you need to use an index but also an array marker: `[]`.
 
-| elements  | objects | array/repeated field  |
-|---|---|---|
-| need path  | need dots to mark nested structure |  need index/array-wildcards to refer to specific position |
-| `id`  | `title.subtitle` | `author.*.firstName` |
-| `name`  | `very.nested.element` | `my.color.2` |
```

Comment:
Is this too confusing? Wanted it as an overview for different scenarios on how to  refrence different elements types.